client/keys.go

```diff
@@ -13,11 +13,9 @@ import (
 
 // PrivateKey represents a keyless-backed RSA private key.
 type PrivateKey struct {
-	public crypto.PublicKey
-	ski    gokeyless.SKI
-	digest gokeyless.Digest
-
-	client *Client
+	public   crypto.PublicKey
+	client   *Client
+	template *gokeyless.Operation
```

Comment:
do you think "opTemplate" would be better? Naming is hard, and  English is my 2nd language. :cry: